javascript/examples/rollup/rollup.config.js

```diff
@@ -0,0 +1,100 @@
+import alias from "@rollup/plugin-alias"
+import nodeResolve from "@rollup/plugin-node-resolve"
+import replace from "@rollup/plugin-replace"
+import wasm from "@rollup/plugin-wasm"
+import { defineConfig } from "rollup"
+import tla from "rollup-plugin-tla"
+
+const inPageConfig = {
+  input: "src/index.js",
+  output: {
+    file: "public/main.js",
+  },
+  // Fixes the `this` instance at top level in automerge's `stable.js` file
+  context: "globalThis",
+  plugins: [
+    // Force automerge-wasm to resolve to its web version
+    alias({
+      entries: [
+        {
+          find: "@automerge/automerge-wasm",
+          replacement: "node_modules/@automerge/automerge-wasm/web/index.js",
+        },
+      ],
+    }),
```

Comment:
Rollup (well, its node-resolve plugin), would try to load the `bundler/automerge_wasm.js` of `@automerge/automerge-wasm`. I tried amending its `__wbg_set_wasm(wasm);` to `__wbg_setwasm(await wasm.default())` (similarly to the replacement of the `initSync` call for `web/index.js`). This is a step towards complying with `@rollup/plugin-wasm` way of handling the wasm module, but it seems there's another part of the code that needs an `await` but I'm not sure which: when running in the browser, an Error gets thrown that `__wbindgen_add_to_stack_pointer` is not a function.

Thanks to it synchronous `initSync` call the `web/index.js` file make things a bit smoother with Rollup it seemes.